.github/workflows/update_credential_provider_script.yml

```diff
@@ -1,6 +1,9 @@
 name: Update credential provider script
 
-on: workflow_dispatch
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
I'm very rusty at this type of scripting so I could easily be wrong, but this looks like it will run everyday.

Reply:
Ah, that's the plan for now until the scripts are stabilized and then we can make it once every two weeks or maybe once a month. I see the confusion might come from the diagram (as it says week). Let me update the diagram.